metrics/README.md

```diff
@@ -0,0 +1,62 @@
+# Clear Containers metrics tests
+
+This is a collection of scripts designed to measure performance and responsiveness
+of the different components which form Clear Containers.
+
+## Metrics tests - content
+
+The tests are organized in the following sections:
+
+| Section   | Short description
+| ----------| --------------------------------------------------------------------------------
+| density   | Measure the amount of memory used by N containers under different configurations.
+| network   | Measure a set of network essentials such as: bandwidth, jitter, latency etc.
+| storage   | Measure storage bandwith using different configurations.
+| time      | Measure the latency that a container[s] take[s] to do a defined workload.
+
+## HOWTO execute metrics scripts
+
+The metrics directory includes a helper script `run_all_metrics.sh`. It allows to execute all metrics
```

Comment:
We are missing a subject in the second sentence:
This helper script allows you to execute either all metrics tests together, or a specific section at time.